docs/docs-beta/docs/dagster-plus/access/rbac/users.md

```diff
@@ -1,7 +1,88 @@
 ---
-title: "User management"
+title: "Managing users in Dagster+"
 displayed_sidebar: "dagsterPlus"
-sidebar_position: 1
+sidebar_label: "User management"
+sidebar_position: 10
 ---
 
-# User management in Dagster+
\ No newline at end of file
+Dagster+ allows you to grant specific permissions to your organization's users, ensuring that Dagster users have access only to what they require.
+
+In this guide, you'll learn how to manage users and their permissions using the Dagster+ UI.
+
+<details>
+<summary>Prerequisites</summary>
+
+- A Dagster+ account
+- The required [Dagster+ permissions](/todo):
+   - **Organization Admins** can add, manage, and remove users
+   - **Admins** can add users
+
+</details>
+
+## Before you start
+
+- **If System for Cross-domain Identity Management specification (SCIM) provisioning is enabled,** you'll need to add new users in your identity provider (IdP). Adding users will be disabled in Dagster+.
+- **If using Google for Single sign-on (SSO)**, users must be added in Dagster+ before they can log in.
```

Comment:
I put these here and not in the **Prereqs** because I didn't want them to get lost. Thoughts?

Reply:
Might be neat as a little info admonition? But I think it's fine. 

Is there a link to learning more about how to implement SCIM?

Reply:
There will be!